codex-rs/core/src/codex.rs

```diff
@@ -861,11 +862,44 @@ impl Session {
         .await
     }
 
+    pub(crate) async fn persist_command_allow_prefix(
```

Comment:
Add a docstring to clarify that this function does two things:

- Amends the in-memory execpolicy to allow the new prefix.
- Amends the on-disk execpolicy so new conversations will also allow the new prefix, by default.